### ISSUE TITLE
[UWP][Accessibility] Incorrect name is defined for all json files at left panel.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3834]
# Platform

* UWP

# Version of SDK

master

# Details

Repro Steps:
1. Launch the application.
2. New screen starts appearing. 
3. Navigate to Left pane JSON files.​
4. Navigate to various json file like Activity Update.JSON, Agenda.JSON etc. at left panel.
5. Verify name of json file like Activity Update.JSON, Agenda.JSON etc. at left panel.

Actual Result: 
Incorrect name = "AdaptiveCardVisualizer.ViewModel.DocumentViewModel" is defined for all json files like Activity Update.JSON, Agenda.JSON etc. at left panel which creates confusion for the screen reader user as they won't be able to proceed further.

Expected Result: 
Correct name should be defined for all json files which is visually appearing like Activity Update.JSON, Agenda.JSON etc. at left panel so that screen reader user could easily proceed with further files.
